.github/workflows/benchmarks.yml

```diff
@@ -55,6 +55,10 @@ jobs:
         override: true
 
     - uses: Swatinem/rust-cache@v2
+
+    - name: Cargo check
```

Comment:
We can also probably remove the cargo checks from the benchmarks workflow, just having it in `test` should be enough

Reply:
Done!